webrtc/RTCPeerConnection-helper.js

```diff
@@ -242,8 +242,7 @@ async function waitForIceGatheringState(pc, wantedStates) {
 async function listenForSSRCs(t, receiver) {
   while (true) {
     const ssrcs = receiver.getSynchronizationSources();
-    assert_true(Array.isArray(ssrcs));
-    if (ssrcs.length > 0) {
+    if (Array.isArray(ssrcs) && ssrcs.length > 0) {
```

Comment:
Wouldn't this just replace spam with timeouts? Throwing an error immediately seems preferable.

Also, if we're removing an assert, there should perhaps be at least one test that contains it.

Reply:
well, this is an assert in a helper function. I do not think they make sense and this one is called in a loop, potentially wreaking havoc to the new "show a list of asserts that passed" functionality

Reply:
Agree, and throwing instead should cause the test to abort.